packages/fiori/src/Wizard.js

```diff
@@ -0,0 +1,650 @@
+
+	get phoneMode() {
+		return this.phoneSize || isPhone();
+	}
+
+	get navAriaRoleDescription() {
+		return this.i18nBundle.getText(WIZARD_NAV_ARIA_ROLE_DESCRIPTION);
+	}
+
+	get navStepDefaultHeading() {
+		return this.i18nBundle.getText(WIZARD_NAV_STEP_DEFAULT_HEADING);
+	}
+
+	/**
+	 * Returns an array of data objects, based on the user defined steps
+	 * to later build the steps (tabs) within the header.
+	 * @returns {Array<Object>}
+	 * @private
+	 */
+	getStepInfo() {
+		const lastEnabledStepIndex = this.getLastEnabledStepIndex();
+
+		return this.steps.map((step, idx, arr) => {
+			const pos = idx + 1;
+
+			// Hide separator if:
+			// (1) its size is under the phone breakpoint
+			// (2) it's the last step and it's not a branching one
+			const hideSeparator = this.phoneMode || ((idx === arr.length - 1) && !step.branching);
+
+			// Calculate the step's aria-roledectioption: "1. heading" or "Step 1".
+			const roleDescription = step.heading ? `${pos}. ${step.heading}` : `${this.navStepDefaultHeading} ${pos}`;
+
+			return {
+				icon: step.icon,
+				heading: this.phoneMode ? "" : step.heading,
+				subheading: this.phoneMode ? "" : step.subheading,
+				initials: pos,
+				selected: step.selected,
+				disabled: step.disabled,
+				hideSeparator,
+				activeSeparator: idx < lastEnabledStepIndex,
+				branchingSeparator: step.branching,
+				pos,
+				size: arr.length,
+				roleDescription,
+				ariaLabel: getEffectiveAriaLabelText(step),
+				refStepId: step._id,
+				tabIndex: this.selectedStepIndex === idx ? "0" : "-1",
+			};
+		});
+	}
+
+	getSelectedStepIndex() {
+		if (this.selectedStep) {
+			return this.slottedSteps.indexOf(this.selectedStep);
+		}
+		return 0;
+	}
+
+	getLastEnabledStepIndex() {
+		const enabledSteps = this.enabledSteps;
+
+		if (enabledSteps.length) {
+			return enabledSteps.length - 1;
+		}
+
+		return 0;
+	}
+
+	getStepByRefId(refId) {
+		return this.slottedSteps.find(step => step._id === refId);
+	}
+
+	getStepWrapperByRefId(refId) {
+		return this.shadowRoot.querySelector(`[data-ui5-content-item-ref-id=${refId}]`);
+	}
+
+	/**
+	 * Determines the closest step index by given scroll position.
+	 *
+	 * @param {Integer} scrollPos scroll position
+	 * @returns {Integer} closestStepIndex the closest step index
+	 * @private
+	 */
+	getClosestStepByScrollPos(scrollPos) {
+		// If the scroll position is found in the scroll offset storage,
+		// the closest step has index bigger than the index of the found scroll offset by 1.
+		if (this.stepScrollOffsets.indexOf(scrollPos) !== -1) {
+			return this.stepScrollOffsets.indexOf(scrollPos) + 1;
+		}
+
+		// Continue searching the closest step index by:
+		// (1) adding the current scroll position to the <code>stepScrollOffsets</code> array
+		// (2) sorting the <code>stepScrollOffsets</code> array
+		// (3) the index of the scroll position gives the index of closest step
+		return [...this.stepScrollOffsets, scrollPos].sort(this.sortAscending).indexOf(scrollPos);
```

Comment:
I think you don't need to pass a compare function here, because this is the default implementation anyway.

Reply:
The default sort order is ascending, but built upon converting the elements into strings and then comparing:

[3, 50, 5, 0, 100, -1].sort() -> [-1, 0, 100, 3, 5, 50]